Cross compile error

When cross compiling to windows, I get the following error : 

```
2016/10/27 10:31:21 apiauth: Can not load timezone Etc/GMT: open /usr/local/go\lib\time\zoneinfo.zip: The system cannot find the path specified.
```

With the full panic stack trace below : 

```
panic: apiauth: Can not load timezone Etc/GMT: open /usr/local/go\lib\time\zoneinfo.zip: The system cannot find the path specified.

goroutine 1 [running]:
panic(0x826ee0, 0xc042116ec0)
        /usr/local/go/src/runtime/panic.go:500 +0x1af
log.Panic(0xc042099ec8, 0x2, 0x2)
        /usr/local/go/src/log/log.go:320 +0xd0
vendor/github.com/pd/apiauth.init.1()
        /<PROJECT_DIR>/src/vendor/github.com/pd/apiauth/apiauth.go:19 +0x15f
vendor/github.com/pd/apiauth.init()
        /<PROJECT_DIR>/src/vendor/github.com/pd/apiauth/apiauth.go:189 +0x5d
main.init()
        /<PROJECT_DIR>/<API_AUTH_CREATION
```

I've removed the bits concerning my own code : It does nothing fancy, just `import github.com/pd/apiauth`. The panic doesn't seem to happen in a function call, it's on module loading.

Can't test yet if it's just a cross-compile error or if it happens on every windows build.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - 1.5
   - 1.6
+  - 1.7
   - tip
 
 script:

--- a/apiauth.go
+++ b/apiauth.go
@@ -16,7 +16,7 @@ var gmt *time.Location
 func init() {
 	loc, err := time.LoadLocation("Etc/GMT")
 	if err != nil {
-		log.Panic("apiauth: Can not load timezone Etc/GMT: ", err)
+		log.Fatalf("Failed to load apiauth -  Can not load timezone Etc/GMT: %s. See https://golang.org/pkg/time/#LoadLocation", err.Error())
 	}
 	gmt = loc
 }

